### PR TITLE
Add usage guide. Fixes #3

### LINKED ISSUE #3: Add --help or -h option
Add an option of **help**, you can put a --help command that can show all the arguments and a description

### PR DESCRIPTION
**Pull request by nacastroc**
### Description

Adds usage guide via `-h`, `--help` CLI options. Closes #3.

### Dependencies

Usage guide display formatting on the console is achieved via the [command-line-usage](https://www.npmjs.com/package/command-line-usage) package.

Snake casing is achieved now via [lodash](https://lodash.com) package, eliminating the usage of the [casing](https://www.npmjs.com/package/casing) package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,14 +27,14 @@ class Province {
 
 | Option               | Alias             | Description                                                         |
 | -------------------- | ----------------- | ------------------------------------------------------------------- |
-| model                | m                 | The model's name you want to export                                 |
-| extension (optional) | e                 | File extension to save. Default `.ts`. Supported extension (`ts`, `js`) |
-| path (optional)      | p, sequelize-path | Path to the index.js sequelize file. By default `./models/index.js` |
+| model                | m                 | The name of the model you want to export                                 |
+| extension (optional) | e                 | File/language extension for the output class file. Defaults to `.ts`. Supports `ts`, `js`, `py` and `cs`. |
```
Comment (orozco92) [inline README.md]: Add php support

File: README.md [hunk 2]
```diff
@@ -27,14 +27,14 @@ class Province {
 
 | Option               | Alias             | Description                                                         |
```
Comment (orozco92) [inline README.md]: Please add help option

File: lib/services/help.service.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        },
+        {
+            header: 'Optional Arguments',
+            optionList: [
+                {
+                    name: 'extension',
+                    alias: 'e',
+                    type: String,
+                    typeLabel: '{underline ts}',
+                    description: 'File/language extension for the output class file.',
```
Comment (orozco92) [inline lib/services/help.service.js]: Please add extension support

File: lib/services/help.service.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+const cliUsage = require('command-line-usage');
+
+function showUsageGuide() {
+    const sections = [
```
Comment (orozco92) [inline lib/services/help.service.js:4]: It would be nice to have an example section.

**Review by orozco92 — CHANGES_REQUESTED** [review]

**Review by luismi930213 — APPROVED** [review]

**Review by orozco92 — APPROVED** [review]

**Pull request merged**